application/event_receiver/requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
Where is requests used? If it isn't can it be removed please

Reply:
done